docs/_docs/04_reference/cli-options.md

```diff
@@ -771,7 +773,7 @@ prefix `--terragrunt-` (e.g., `--terragrunt-config`). The currently available op
   - [terragrunt-modules-that-include](#terragrunt-modules-that-include)
   - [terragrunt-fetch-dependency-output-from-state](#terragrunt-fetch-dependency-output-from-state)
   - [terragrunt-use-partial-parse-config-cache](#terragrunt-use-partial-parse-config-cache)
-  - [terragrunt-include-module-prefix](#terragrunt-include-module-prefix)
+  - [terragrunt-include-module-prefix (DEPRECATED: use terragrunt-raw-module-output)](#terragrunt-include-module-prefix)
```

Comment:
Same here.

Reply:
Na, I was saying I think this is better as two links. You can see the md [here](https://github.com/gruntwork-io/terragrunt/pull/3338#discussion_r1731690318). It's not worth changing now, though. I'll look to adjust it later.

Reply:
Ah, roger. Fixed.